### PR TITLE
BZ2050710 Remove SHA 1 fingerprint of vCenter host prerequisite for 2.3

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
BZ2050710 Remove SHA 1 fingerprint of vCenter host prerequisite

MTV 2.3

https://bugzilla.redhat.com/show_bug.cgi?id=2050710

### CODE REVIEW

**Comment by istein1** [comment]
LGTM

File: documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc [hunk 1]
```diff
@@ -113,10 +113,12 @@ You can migrate virtual machines to {virt} from the command line.
 
 [IMPORTANT]
 ====
-*  You must be logged in as a user with the {ocp} `cluster-admin` privileges.
+*  You must be logged in as a user with `cluster-admin` privileges.
 
 *  VMware only: You must have the minimal set of xref:vmware-privileges_{context}[VMware privileges].
 
+*  VMware only: You must obtain the xref:obtaining-vmware-fingerprint_{context}[vCenter SHA-1 fingerprint].
```
Comment (stoobie) [inline documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:120]: @sbeskin-redhat Please change to *You must have*.

I think that we as a team need to have a discussion about style guidelines for Prerequisites. There was a big discussion in the style council about phrasing prerequisites as something you need to do vs an existing condition. For example:
*You must **obtain** the xref:obtaining-vmware-fingerprint_{context}[vCenter SHA-1 fingerprint].*
vs
*You must **have** the xref:obtaining-vmware-fingerprint_{context}[vCenter SHA-1 fingerprint].*

The decision was to "Avoid using imperative formations." and part of the idea there is that if we're telling the user to do something (like obtain), then we're actually adding a step to the procedure in the prereqs.

See https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites

**Review by stoobie — CHANGES_REQUESTED** [review]
One tiny comment. Otherwise LGTM.

File: documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc]: Done

**Review by stoobie — APPROVED** [review]
LGTM

**Pull request merged**